MetaMask/metamask-extension
# UX: Update querystring for analytics

## **Description**

Fixes the querystring for the analytics Iframe to the portfolio.

## **Manual testing steps**

1.  Go through on boarding
2. Ensure that the iframe on the Create Password page has correct querystring of `?env=production&mmi={SOMESTRING}`


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
